tests/contrib/autoguide/test_advi.py

```diff
@@ -179,6 +186,7 @@ def model():
     AutoDiagonalNormal,
     AutoMultivariateNormal,
     AutoLowRankMultivariateNormal,
+    AutoIAFNormal,
```

Comment:
wasn't this to be removed?

are there any "actual" tests of autoiafnormal here?

Reply:
why would this be removed?

Reply:
why is that needed for testing parallel enumeration?